packages/debug/package.json

```diff
@@ -67,7 +67,8 @@
     "test": "theiaext test"
   },
   "devDependencies": {
-    "@theia/ext-scripts": "^0.14.0"
+    "@theia/ext-scripts": "^0.14.0",
+    "colors": "^1.4.0"
```

Comment:
left overs? Where is it used?

Please keep dev dependencies in root package to ensure that we don't use many different versions in the repo.

Reply:
It used for tests, will move at the root.